docs/guidelines-for-reviewing-datascience-code.md

```diff
@@ -1,25 +1,23 @@
-# Guidelines for Reviewing Data Science Code
+# 10 Principles for Reviewing Data Science Code
 
-Code reviews are common practice in the software engineering field and have been for some time. They provide developers with a well defined approach to sharing knowledge, ensuring exposure to new ideas and technologies, enforcing quality assurance of code and standards for the engineering process across both the code base and the team itself.  As members of the AI CoE in the office of the CTO, we believe that having a well defined approach for how to do the same thing with data science and machine learning code is critical. Although, code is code, and we approach data science work as a subset of software engineering, there are some data science specific challenges associated with the code review process that should be addressed. Below are the 11 guidelines we use in our group to review data science work.     
+Code reviews are common practice in the software engineering field and have been for some time. They provide developers with a well-defined approach to sharing knowledge, ensuring exposure to new ideas and technologies, enforcing quality assurance of code and standards for the engineering process across both the codebase and the team itself. As members of the AI CoE in the office of the CTO, we believe that having a well-defined approach for how to do the same thing with data science and machine learning code is critical. Although, code is code, and we approach data science work as a subset of software engineering, there are some data science specific challenges associated with the code review process that should be addressed. Below are the 10 principles we use in our group to review data science work.
 
-1. **Understand the PR** : When asked for a review, make sure to first understand the issue linked and the goal of the pull request.
+1. **Understand the PR** : When asked for a review, make sure to first understand the issue linked and the goal of the pull request. Go through the Pull Request description to understand what has changed, a checklist of prerequisites like added documentation, notebooks, etc., and any additional notes the reviewer should be aware of.
```

Comment:
```suggestion
1. **Understand the Context** : Make sure you read the pull request description and any linked issues so you know the data scientist's goal. This will help you provide appropriate level of feedback and focus on the meaningful changes. 
```